packages/contracts-ops/README.md

```diff
@@ -2,6 +2,31 @@
 
 Contains forge scripts for Nomad system maintenance
 
+## Building Blocks
+
+- `Config.sol` utilities for loading and parsing Nomad Config JSON
+  - initalize the config with the path to a JSON config
+- `CallBatch.sol` utility for aggregating calls into a callbatch, and
```

Comment:
could be useful to include a bit more contextual information about what a callbatch is. even "into a governance callbatch" could be helpful

Reply:
I think the appropriate place for that is the governance documentation, not the toolbox